<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>display</class>
 <widget class="QWidget" name="display">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>10</y>
     <width>600</width>
     <height>400</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>600</width>
     <height>400</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="nextBut">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Meiryo UI</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prevBut">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Meiryo UI</family>
    </font>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
  </widget>
  <widget class="QLabel" name="rankText">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Meiryo UI</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Image %d/%d: Ranked %.03f/9 || set %d/%d</string>
   </property>
  </widget>
  <widget class="QSlider" name="slider">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>601</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="prevSetBut">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Meiryo UI</family>
    </font>
   </property>
   <property name="text">
    <string>Prev. In Set</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextSetBut">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>470</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Meiryo UI</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Next In Set</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
